### DISCUSSION TITLE
How to avoid a dependent resource to be created when manually created in the cluster?

### DISCUSSION THREAD

**Discussion by mfvitale** [operator-framework/java-operator-sdk#2570] [Q&A] [answered]
What is the normal approach to manage this kind of use case? I want that the depended resource is created only if not already exiting (created manually).

**Comment by metacosm** [comment, marked as answer]
You usually only need to define the appropriate `desired` method implementation. If a resource matching the desired state already exists on the cluster, then the SDK won't be attempting to create it.
